src/Crypto/Spake2/Groups/Ed25519.hs

```diff
@@ -340,7 +340,11 @@ That final check is a Python assertion,
 which would crash the program if incorrect.
 For programming convenience, I just skip these values.
 
-jml doesn't know what is meant by the 'order' of a point.
+The 'order' of a point \(x\( is the number \(n\) such that:
```

Comment:
The "Note" thing is a GHC convention that I've recently started to experiment with. See https://www.stackbuilders.com/news/the-notes-of-ghc